src/pages/auth/signin/EmailLoginPage.tsx

```
@@ -0,0 +1,17 @@
+import { EmailLoginForm } from "@/components/auth/signin/EmailLoginForm";
+
+type Props = {};
+
+export default function EmailLoginPage({}: Props) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)